ui/plugins/controller.js

```diff
@@ -88,7 +93,11 @@ treeherder.controller('PluginCtrl', [
                     $scope.eta_abs = Math.abs($scope.job.get_current_eta());
                     $scope.typical_eta = $scope.job.get_typical_eta();
                     $scope.jobRevision = ThResultSetStore.getSelectedJob($scope.repoName).job.revision
-
+                    $scope.job_ids = results[4];
+                    console.log(results);
+                    _.forEach($scope.job_ids, function(job) {
+                        console.log(job[0].value);
+                    });
```

Comment:
> So looking at the json I see at least signature 32206df28ab3b52e15d26f73178ee78e2af7e760 which corresponds to ts_paint. 

yes, we can get corresponds signature to ts_paint, but can't distinguish it with other signature from a bunch of data. Because the job value of  32206df28ab3b52e15d26f73178ee78e2af7e760 is **1132.00** but the value we want for ts_paint is 1144.70. 

However, let me sort the difficult I meet, maybe it can help us set it ASAP :)
The fist step I do is get a bunch of signature object by job_id.  Then I unpack the job_detail object and pick up the job we want to display(I assume we only need to display the 'talos result' job). So we can find the value for each job(eg. 677.65 for a11yr_paint, 2199.78 for sessionrestore)

At last, we can compare each job value in signature object with we found in job detail. If we got two *same* value, then we can assume they are same job and could be used to generate a link to perherder.

**So, some difficulties I have:**
**1) we can't convert some value like 'MB' to pure raw data (pass)**
**2) I don't know the correct way to pick up the right signature from signature list. Although, we can get the correct signature list. Otherwise, we need to display all the signature we got by job id.**
**ps**: I always found pick up data by compare job value is somehow inaccuracy. maybe it's a **wrong** way to deal with this issue. and maybe we should just looking up each signature and display them like: 

suite: url to perf 
rather than 
testName: value(url to perf)

But there are too many things we need to display which isn't right also, I think

>So if you're not seeing it in your output, there must be a bug in the way you're formatting things. Let me know if you need help tracking this down, but I suspect it's a pretty simple problem. :)

Yes, I just feel like jump into a 'cul-de-sac' and need to some hits to get out. always wonder if the method I use is right and can't figure out the right way to do it either. Great appreciate if you can help me out! Thank you!